plugins/woocommerce-blocks/assets/js/blocks/product-filters-overlay-navigation/style.scss

```diff
@@ -0,0 +1,16 @@
+.wc-block-product-filters-overlay-navigation__wrapper {
+	display: flex;
+}
+
+.wc-block-product-filters-overlay-navigation {
+	display: inline-flex !important;
```

Comment:
Since this also effects the frontend, we should try to avoid using `!important` as much as possible. Does this need to be `inline-flex`? Will `flex` not work here? If you really need it to be `inline-flex`, you can just add specificity like `.is-layout-flex.wc-block-product-filters-overlay-navigation { display: inline-flex; }`. This way it will be easier for themes to override this.

Reply:
Great catch! I check it again and we don't need inline-flex. It's left from my previous implementation. I removed it in 655aa15, thanks a lot!